### PR TITLE
Setting of Routes Skeletons (REST API)  #39523

### PR DESCRIPTION
**Pull request by Roshan-successive**
- Learn about REST APIs and Postman.
- Overview of Route.
- Defining skeleton of every routes.

### CODE REVIEW

File: src/Server.ts [hunk 1]
```diff
@@ -24,7 +34,7 @@ console.log(err);
 }
 console.log(`App is running on port ${PORT}`);
 // tslint:disable-next-line: semicolon
```
Comment (suraj-bha) [inline src/Server.ts:35]: Remove this lint rule disable.

File: src/controllers/trainee/controller.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        }
+    }
+
+    create(req, res, next) {
+        try {
+            var console: Console
+            console.log("Inside post method of Trainee Controller");
+
+            res.send({
+                message: "Trainee fetched successfully",
```
Comment (suraj-bha) [inline src/controllers/trainee/controller.ts]: update the message as per the operation

File: src/controllers/trainee/controller.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        }
+    }
+
+    update(req, res, next) {
+        try {
+            var console: Console
+            console.log("Inside update method of Trainee Controller");
+
+            res.send({
+                message: "Trainee fetched successfully",
```
Comment (suraj-bha) [inline src/controllers/trainee/controller.ts]: update the message as per the operation

File: src/controllers/trainee/controller.ts [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+            console.log("Inside err", err);
+        }
+    }
+    delete(req, res, next) {
+        try {
+            var console: Console
+            console.log("Inside delete method of Trainee Controller");
+
+            res.send({
+                message: "Trainee fetched successfully",
```
Comment (suraj-bha) [inline src/controllers/trainee/controller.ts]: update the message as per the operation

File: src/Server.ts (hunk 1 above)
Comment (Roshan-successive) [reply src/Server.ts:35]: By mistake i forget to remove this lint rule but i resloved  it.

File: src/controllers/trainee/controller.ts (hunk 2 above)
Comment (Roshan-successive) [reply src/controllers/trainee/controller.ts]: Updated all the message as per the operation

**Review by suraj-bha — APPROVED** [review]

**Pull request merged**